cmd/tk/export.go

```diff
@@ -70,11 +73,17 @@ func exportCmd() *cli.Command {
 			if err := tmpl.Execute(&buf, m); err != nil {
 				log.Fatalln("executing name template:", err)
 			}
+
 			name := strings.Replace(buf.String(), "/", "-", -1)
+			name = strings.Replace(name, string(rune(7)), "/", -1)
```

Comment:
I like the idea of `rune(7)` here! Could you please put this in a constant with a descriptive name, so it feels a bit less "magic"?